src/Storage/Listener/AbstractListener.php

```diff
@@ -6,17 +6,17 @@
  */
 namespace Burzum\FileStorage\Storage\Listener;
 
+use Burzum\FileStorage\Storage\PathBuilder\PathBuilderTrait;
 use Burzum\FileStorage\Storage\StorageTrait;
 use Burzum\FileStorage\Storage\StorageUtils;
 use Cake\Core\InstanceConfigTrait;
 use Cake\Datasource\EntityInterface;
 use Cake\Event\Event;
 use Cake\Event\EventListenerInterface;
+use Cake\Filesystem\Folder;
```

Comment:
@burzum its used [here](https://github.com/robertpustulka/cakephp-file-storage/blob/3.0-path-builder-trait/src/Storage/Listener/AbstractListener.php#L300) and its just the result of alphabetical reorder of use statements.